### ISSUE TITLE
Update node.js to latest version

### ISSUE THREAD

**Issue by pdawyndt** [dodona-edu/universal-judge#504]
Student asked whether JS could be updated to latest version.

**Comment by bmesuere** [comment]
This is probably about node.js and not JavaScript. There are however very few language changes between node 18 (currently used) and node 20 (current stable) so it is not clear why the student would want an update. I suggest waiting till node 22 (out in 10 days) or till we update other dependencies.

```
- String.prototype.isWellFormed and toWellFormed: These methods are particularly useful for ensuring proper string format. This help ensure that user string input is in the correct UTF-16 format, reducing errors in the age of emojis.
- Methods that change Array and TypedArray by copy: This can be useful for creating modified copies of arrays without affecting the original data, which is especially relevant in cases where you need to preserve the original data for comparison or other purposes.
- Resizable ArrayBuffer and growable SharedArrayBuffer: provides greater flexibility, allowing for more efficient memory allocation.
- RegExp v flag with set notation and properties of strings: adds functionality for regular expressions.
- WebAssembly tail call: provides a way to optimize certain types of function calls.
```

**Comment by pdawyndt** [comment]
Missing language feature observed by student: `Arrays.prototype.toSorted() ` (supported since node.js 20.0.0).

**Comment by niknetniko** [comment]
Fixed by #511
